bolt-sidecar/src/driver.rs

```diff
@@ -78,11 +89,11 @@ impl SidecarDriver<StateClient, BlsSigner, PrivateKeySigner> {
     }
 }
 
-impl<C: StateFetcher, BLS: SignerBLS, ECDSA: SignerECDSA> SidecarDriver<C, BLS, ECDSA> {
+impl<C: StateFetcher, ECDSA: SignerECDSA> SidecarDriver<C, ECDSA> {
```

Comment:
I think we can still have the generic here, by changing the bind to the bls async trait:

```rust
impl<C: StateFetcher, BLS: SignerBLSAsync, ECDSA: SignerECDSA> SidecarDriver<C, BLS, ECDSA> {
```